sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/coders/JAXBCoder.java

```diff
@@ -109,6 +126,17 @@ public String getEncodingId() {
     return getJAXBClass().getName();
   }
 
+  private static class CloseIgnoringInputStream extends FilterInputStream {
+    protected CloseIgnoringInputStream(InputStream in) {
+      super(in);
+    }
+
+    @Override
+    public void close() {
+      // Do nothing. JAXB closes the underyling stream so we must filter out those calls.
+
```

Comment:
newline.

Also, I was wrong about flush -- no need to flush an input stream (duh)